docs/pages/machine-id/guides/kubernetes.mdx

```diff
@@ -0,0 +1,244 @@
+---
+title: Machine ID with Kubernetes Access
+description: Accessing Kubernetes clusters via Machine ID
+---
+
+In this guide, we will demonstrate how to use Machine ID to access a Kubernetes
```

Comment:
Same recommendation as the previous guide re: the "from an automated service" phrasing.